js/app.js

```diff
@@ -0,0 +1,100 @@
+//What is the difference between interpolation and concatenation?
+// Interpolation uses a variable to represent a value.
+//Concatenation combines two 'strings' together.
+
+//Give an example of each.
+// Interpolation:
+//// const verb = 'believe'
+// const noun = 'cheese'
+// console.log("Please " + verb + " the " + noun + ".")
+//
+//Concatenation:
+//const cityName = 'Astoria'
+//const weather = 'rainy'
+//console.log(cityName + " is " + weather + " this time of year.");
+//**ARE ALL CONCATENATIONS INTERPOLATIONS BUT NOT ALL INTERPOLATIONS ARE CONCATENATIONS??**
```

Comment:
neither is true.
not all concatenations are interpolations (i.e. "bleu " + "cheese") is not an interpolation
not all interpolations are concatenations: e.g. console.log(myFaveTypeOfCheese) is not a concatenation
lmk if you have any questions.